Documentation/configuring-hive-metastore.md

```diff
@@ -30,25 +30,29 @@ Use [metastore-storage.yaml][metastore-storage-config] as a template and adjust
 
 - `spec.hive.spec.metastore.storage.size`
 
-## Use MySQL for the Hive Metastore database
+### Use MySQL for the Hive Metastore database
 
 By default to make installation easier Metering configures Hive to use an embedded Java database called [Derby](https://db.apache.org/derby/#What+is+Apache+Derby%3F), however this is unsuited for larger environments or metering installations with a lot of reports and metrics being collected.
 Currently two alternative options are available, MySQL and PostgreSQL, both of which have been tested with the metering-operator.
 
-There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , `username` , and `password`.
+There are three configuration options you can use to control the database used by Hive metastore: `url` , `driver` , and `secretName` .
+The `url` is the url of the MySQL instance, port followed by the database name.
+The `driver` configures the javax.jdo.option.ConnectionDriverName, which is the class name for the jdbc driver that will be used to store hive metadata.
+The `secretName` is the name assigned to the secret which contains the base64 encrypted username and password for the database.
 
-Using MySQL:
+### Using MySQL
+
+To use MySQL create a secret in your namespace and then add the following to your Metering CR under the top level `spec`:
 
 ```
-spec:
-  hive:
-    spec:
-      config:
-        db:
-          url: "jdbc:mysql://mysql.example.com:3306/hive_metastore"
-          driver: "com.mysql.jdbc.Driver"
-          username: "REPLACEME"
-          password: "REPLACEME"
```

Comment:
Let's revert back to the original example's YAML hierarchy, but still continue to replace the `username` and `password` fields with `secretName`.

Reply:
In the CR it is only a secretName,  so the above is a better matching example though. 
Unless you have other ideas for why un and pw should be there?

Reply:
No, I meant the current example is mis-aligned compared to the original. We want to be removing the `username` and `password` references.

```yaml
spec:
  hive:
    spec:
      config:
        db:
          url: "jdbc:mysql://mysql.example.com:3306/hive_metastore"
          driver: "com.mysql.jdbc.Driver"
          secretName: "REPLACEME"
```